tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -53,6 +53,16 @@ def init_sanity(self):
         """
         self.sanity_helpers = Sanity()
 
+    @pytest.fixture(autouse=True)
+    def health_checker(self):
+        try:
+            status = ceph_health_check_base()
+            if status:
+                logger.info("Health check passed")
+        except CephHealthException:
```

Comment:
We are losing some information here from the original exception. Since the exception message is `f"Ceph cluster health is not OK. Health: {health}"` can we simply use that as the skip message? This way when we skip this test we know what health state the cluster is in.

We can do something like this:
```python
except CephHealthException as e:
    # skip because ceph is not in good health
    pytest.skip(e)
```

Reply:
Done